subsys/bluetooth/controller/ll_sw/nordic/lll/lll_scan_aux.c

```diff
@@ -470,6 +470,9 @@ static int prepare_cb(struct lll_prepare_param *p)
 	/* Initialize scanning state */
 	lll_aux->state = 0U;
 
+	/* Reset Scan context association with Aux context as ULL scheduling used here */
+	lll->lll_aux = NULL;
+
```

Comment:
This code also has a race and is causing the LLL scheduled auxiliary PDU reception to leak aux contexts :-(